### PR TITLE
minor: fixed hierarchy and added a table of content

### PR DESCRIPTION
**Pull request by pm47**
I spotted some inconsistencies in naming between channel establishment and channel closing.

I'd like to do:
- `shutdown` -> `close_channel`
- `close_signature` -> `closing_signed`

what do you think?

### CODE REVIEW

File: 02-peer-protocol.md [hunk 1]
```diff
@@ -11,9 +11,25 @@ The messages described in this document are grouped logically into 4 groups by t
  - Routing (types `256`-`511`): node and channel announcements, as well as any active route exploration.
 
 # Table of Contents
-
-FIXME
-
+  * [Channel](#channel)
```
Comment (Roasbeef) [inline 02-peer-protocol.md]: Nice addition with the table of contents! 👍

Comment (rustyrussell) [inline 02-peer-protocol.md]: Hmm, there's shutdown, which is where we don't add anything new, and closing, which is where we're negotiation the final close tx.  I used to call shutdown "clearing" but that has a specific finance meaning which was deeply sucky.

Ah, I see you reverted that AFAICT?  I'll squash and merge.

**Comment by rustyrussell** [comment]
Squashed into three patches, and merged.  Thanks!

File: 02-peer-protocol.md (hunk 1 above)
Comment (pm47) [inline 02-peer-protocol.md]: > Ah, I see you reverted that AFAICT? I'll squash and merge.

Yeah, I didn't want to mix two separate topics in the same PR. I am just OCDing on `funding_signed` vs `close_signature`  😵

**Comment by rustyrussell** [comment]
Pierre-Marie Padiou <notifications@github.com> writes:
> pm47 commented on this pull request.
>
>
>
>> @@ -11,9 +11,25 @@ The messages described in this document are grouped logically into 4 groups by t
>   - Routing (types `256`-`511`): node and channel announcements, as well as any active route exploration.
>  
>  # Table of Contents
> -
> -FIXME
> -
> +  * [Channel](#channel)
>
>> Ah, I see you reverted that AFAICT? I'll squash and merge.
>
> Yeah, I didn't want to mix two separate topics in the same PR. I am just OCDing on `funding_signed` vs `close_signature`  😵 

Agreed.  Pick one to change and submit another PR :)

Thanks!
Rusty.

**Pull request closed without merging**